### PR TITLE
Small cleanup via codemods

### PR DESCRIPTION
**Pull request by pradeepnschrodinger**
## Description

I'm doing a small cleanup over the codebase via codemods, as part of [LDIDEAS-6126](https://jira.schrodinger.com/browse/LDIDEAS-6126).
There's 3 codemods applied here:
1. [no-vars](https://github.com/cpojer/js-codemod#no-vars) (convert `var` to `let`/`const`)
2. [object-shorthand](https://github.com/cpojer/js-codemod#object-shorthand) (Transforms object literals to use [ES6 shorthand](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015) for properties and methods.)
3. [lodash-imports-codemod](https://gist.github.com/laurenashpole/a7bdc373e97d4c1efd1eea1aa8ee4b0d) (split lodash imports)

Each codemod is applied through a separate commit to see the individual set of changes easier.
I also don't recall encountering any special cases, so the full changes here are exhaustive to the codemods.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by AmanGupta2708** [comment]
why not to write the eslint rule with the fix option?

**Comment by pradeepnschrodinger** [comment]
> What you think about writing the eslint rule with the fix option?

Sorry, not sure I got you. Does eslint have a way run "lint" using these codemods?

**Comment by AmanGupta2708** [comment]
> > What you think about writing the eslint rule with the fix option?
> 
> Sorry, not sure I got you. Does eslint have a way "lint" using these codemods?

Eslint has rule for  [no-var](https://eslint.org/docs/latest/rules/no-var#rule-details) and [object-shorthand](https://eslint.org/docs/latest/rules/object-shorthand#rule-details), and problem identify by them are usually fixable by `eslint --fix`.

**Comment by pradeepnschrodinger** [comment]
> Eslint has rule for [no-var](https://eslint.org/docs/latest/rules/no-var#rule-details) and [object-shorthand](https://eslint.org/docs/latest/rules/object-shorthand#rule-details), and problem identify by them are usually fixable by `eslint --fix`.

Awesome. I can add them in to eslint's config, and have precommit take care of it

File: .eslintrc.yaml [hunk 1]
```diff
@@ -9,3 +9,7 @@ parserOptions:
 
 extends:
   - prettier
+
+rules:
+  no-var: "warn"
+  object-shorthand: "warn"
```
Comment (pradeepnschrodinger) [inline .eslintrc.yaml:15]: Added the `no-var` and `object-shorthand` rules to eslint.

This'll be run by precommit over staged files, and issues (if any) will be fixed automatically.

**Review by pradeepnschrodinger — COMMENTED** [review]
@AmanGupta2708 , I added the rules to eslint as you requested.

File: .eslintrc.yaml (hunk 1 above)
Comment (AmanGupta2708) [inline .eslintrc.yaml:15]: nit: add newline at end of file.

Comment (AmanGupta2708) [reply .eslintrc.yaml:15]: Thanks for doing this !!

**Review by AmanGupta2708 — APPROVED** [review]

**Comment by pradeepnschrodinger** [comment]
Merged locally through https://github.com/schrodinger/fixed-data-table-2/commit/2b36a1dd7931d689512c65fc41563519e0f7b24e.

**Pull request closed without merging**